src/olympia/reviewers/templates/reviewers/includes/files_view.html

```diff
@@ -21,5 +21,28 @@
     {% endif %}
   </span>
 </li>
+{% if waffle.flag('code-manager') %}
+  <li class="code-manager-links">
+    <div>
+      <strong>Code Manager</strong>
+      <img width="25" height="9" src="{{ static('img/reviewers/new.gif') }}">
+    </div>
+    <a
+      href="{{ code_manager_url('/browse/{}/versions/{}/'.format(addon.pk, version.pk)) }}"
```

Comment:
I wonder if this `.format` substitution should be moved down to `code_manager_url` also - i.e. add `addon`, `version` and (optional) `diff` args like `def code_manager_url(addon, version, diff=None)`  (and alter the url from browse to compare based on if diff is `None`)

Reply:
The point of `code_manager_url()` is simply to share the common host prefix logic. I don't see any other shared functionality besides that. 

The two URLs are distinct; they both use add-on IDs and version IDs but in different ways. If I were to move that logic into `code_manager_url()` then it would be implemented with an if statement switching between the two URLs and I don't think that would be helpful. Also, it would have a negative side affect of forcing the reader to inspect `code_manager_url()` to understand what the final URL will look like.